### PR TITLE
WIP: Update logging gapic yaml

### PR DESCRIPTION
**Pull request by michaelbausor**
Updates logging to use resource names

### CODE REVIEW

File: google/logging/v2/logging_gapic.yaml [hunk 1]
```diff
@@ -19,19 +19,52 @@ language_settings:
...
 collection_oneofs:
 - oneof_name: parent_oneof
   collection_names:
   - parent
   - organization
+  - folder
+  - billing
 - oneof_name: sink_oneof
   collection_names:
   - sink
```
Comment (garrettjonesgoogle) [inline google/logging/v2/logging_gapic.yaml]: maybe this should be renamed to `project_sink`?

File: google/logging/v2/logging_gapic.yaml [hunk 2]
```diff
@@ -19,19 +19,52 @@ language_settings:
...
   - organization_sink
+  - folder_sink
+  - billing_sink
 - oneof_name: log_oneof
   collection_names:
   - log
```
Comment (garrettjonesgoogle) [inline google/logging/v2/logging_gapic.yaml]: Maybe rename to `project_log`?

**Comment by garrettjonesgoogle** [comment]
LGTM

**Comment by bjwatson** [comment]
Forgot to remove "WIP" from the commit message. ;)

**Comment by michaelbausor** [comment]
Oops... do you think this is worth amending and force-pushing to update?

**Comment by bjwatson** [comment]
No, let's not force update anything on master. Just pointing out in case anyone is wondering why a WIP was merged ('cause it's not really a WIP anymore).

**Comment by michaelbausor** [comment]
OK no worries!

**Pull request merged**